mak/WINDOWS

```diff
@@ -1202,6 +1202,9 @@ $(IMPDIR)\core\sys\windows\rpcnterr.d : src\core\sys\windows\rpcnterr.d
 
 $(IMPDIR)\core\sys\windows\schannel.d : src\core\sys\windows\schannel.d
 	copy $** $@
+	
```

Comment:
Please remove the tab character from this line. Trailing whitespace is not allowed by CI rules.